src/store/token-dashboard/sagas.ts

```diff
@@ -417,6 +421,11 @@ function* watchForDiscordCode() {
   yield put(setDiscordCode({ code: appended }))
 }
 
+function* preloadWalletProviders() {
+  yield loadWalletConnect()
```

Comment:
do you think this would be better to just do on the /audio page instead of in the modal?

Reply:
I don't think it makes too much of a difference - might be slightly better on the /audio page to help reduce potential lag (depending on internet speed) w/ the web3modal overlay appearing. Will move though.